docs/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Home
+---
+
+This pipeline can be used to process small RNA data (~20-60bp short reads)
+ starting from FastQ files.
+It will perform adapter clipping (using cutadapt), mapping (using bowtie) and expression
+quantification (using HTSeq-Count).
+
+This pipeline is part of [BioWDL](https://biowdl.github.io/)
+developed by the SASC team at [Leiden University Medical Center](https://www.lumc.nl/).
+
+## Usage
+This pipeline can be run using
+[Cromwell](http://cromwell.readthedocs.io/en/stable/):
+```bash
+java -jar cromwell-<version>.jar run -i inputs.json small-rna.wdl
+```
+
+### Dependency requirements and tool versions
+Biowdl pipelines use docker images to ensure  reproducibility. This
+means that biowdl pipelines will run on any system that has docker
+installed. Alternatively they can be run with singularity.
+
+For more advanced configuration of docker or singularity please check
+the [cromwell documentation on containers](
+https://cromwell.readthedocs.io/en/stable/tutorials/Containers/).
+
+Images from [biocontainers](https://biocontainers.pro) are preferred for
+biowdl pipelines. The list of default images for this pipeline can be
+found in the default for the `dockerImages` input.
+
+### Inputs
+Inputs are provided through a JSON file. The minimally required inputs are
+described below, but additional inputs are available.
+A template containing all possible inputs can be generated using
+Womtool as described in the
+[WOMtool documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
+See [this page](/inputs.html) for some additional general notes and information
```

Comment:
This inputs page is not accurate anymore. It still talks about JAR inputs and precommands. We should probably remove it or add a note that the page is only applicable to old versions.